File: src/Entity/Suivi.php
<?php

namespace App\Entity;
use DateTime;
use DateTimeInterface;
use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints as Assert;

/**
 * Suivi
 *
 * @ORM\Table(name="suivi", indexes={@ORM\Index(name="fk_simple_client", columns={"client"}), @ORM\Index(name="fk_psycho_username", columns={"username"})})
 * @ORM\Entity
 */
class Suivi
{
    /**
     * @var int
     *
     * @ORM\Column(name="id_s", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idS;

    /**
     * @var string|null
     * @ORM\Column(name="titre_s", type="string", length=50, nullable=false)
     * @Assert\NotBlank(message=" Adresse devrait etre non vide")
     */
    private $titreS;

    /**
     * @var DateTime
     * @ORM\Column(name="date_ds", type="date", nullable=false)
     * @Assert\DateTime(message="Faut que ca soit une date valide")
     */
    private $dateDs;

    /**
     * @var DateTime
     * @ORM\Column(name="date_fs", type="date", nullable=false)
     * @Assert\DateTime(message="Faut que ca soit une date valide")
     */
    private $dateFs;

    /**
     * @var DateTime
     * @ORM\Column(name="temps_ds", type="time", nullable=false)
     * @Assert\DateTime(message="Faut que ca soit du temps")
     */
    private $tempsDs;

    /**
     * @var DateTime
     * @ORM\Column(name="temps_fs", type="time", nullable=false)
     * @Assert\DateTime(message="Faut que ca soit du temps")
     */
    private $tempsFs;

    /**
     * @var string|null
     * @ORM\Column(name="username", type="string", length=50, nullable=false)
     * @ORM\ManyToOne(targetEntity=Psycho::class,inversedBy="username")
     */
    private $username;

    /**
     * @var String|null
     * @ORM\Column(name="client", type="string", length=20, nullable=false)
     * @Assert\NotBlank(message="Client doit étre existant")
     * @ORM\ManyToOne(targetEntity=Simple::class,inversedBy="username")
     */
    private $client;

    /**
     * @return int
     */
    public function getIdS(): int
    {
        return $this->idS;
    }

    /**
     * @param int $idS
     */
    public function setIdS(int $idS): void
    {
        $this->idS = $idS;
    }

    /**
     * @return string|null
     */
    public function getTitreS(): ?string
    {
        return $this->titreS;
    }

    /**
     * @param string|null $titreS
     */
    public function setTitreS(?string $titreS): void
    {
        $this->titreS = $titreS;
    }

    /**
     * @return DateTimeInterface
     */
    public function getDateDs(): ?DateTimeInterface
    {
        return $this->dateDs;
    }

    /**
     * @param DateTimeInterface $dateDs
     */
    public function setDateDs(DateTimeInterface $dateDs): void
    {
        $this->dateDs = $dateDs;
    }


    public function getDateFs(): ?DateTimeInterface
    {
        return $this->dateFs;
    }


    public function setDateFs(DateTimeInterface $dateFs): void
    {
        $this->dateFs = $dateFs;
    }

    public function getTempsDs(): ?DateTimeInterface
    {
        return $this->tempsDs;
    }

    public function setTempsDs(DateTimeInterface $tempsDs): void
    {
        $this->tempsDs = $tempsDs;
    }

    public function getTempsFs(): ?DateTimeInterface
    {
        return $this->tempsFs;
    }

    public function setTempsFs(DateTimeInterface $tempsFs): void
    {
        $this->tempsFs = $tempsFs;
    }

    /**
     * @return String
     */
    public function getUsername(): String
    {
        return $this->username;
    }

    /**
     * @param String $username
     */
    public function setUsername(String $username): void
    {
        $this->username = $username;
    }

    /**
     * @return String
     */
    public function getClient(): String
    {
        return $this->client;
    }

    /**
     * @param String $client
     */
    public function setClient(String $client): void
    {
        $this->client = $client;
    }

    public function __toString() {
        return $this->getUsername();
    }
}

File: src/entities/Suivi.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entities;
import javafx.scene.control.TextField;
/**
 *
 * @author shidono
 */
public class Suivi {
    private Integer id_s;
    public void setId_s(Integer id_s) {
        this.id_s = id_s;
    }
    private String client;
    private String username;
    private String titre_s;
    private String date_ds;
    private String date_fs;
    private String temps_ds;
    private String temps_fs;

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public Suivi() {
        
    }

    public void setClient(String client) {
        this.client = client;
    }

    public void setTitre_s(String titre_s) {
        this.titre_s = titre_s;
    }

    public void setDate_ds(String date_ds) {
        this.date_ds = date_ds;
    }

    public void setDate_fs(String date_fs) {
        this.date_fs = date_fs;
    }

    public void setTemps_ds(String temps_ds) {
        this.temps_ds = temps_ds;
    }

    public void setTemps_fs(String temps_fs) {
        this.temps_fs = temps_fs;
    }

    public Suivi(int id, String titre, String debut, String fin, String timedeb, String timefin) {
        this.id_s = id;
        this.titre_s = titre;
        this.date_ds = debut;
        this.date_fs = fin;
        this.temps_ds = timedeb;
        this.temps_fs = timefin;
    }

    public Suivi(String client,String titre, String datedeb, String datefin, String timedeb, String timefin) {
        this.client = client;
        this.titre_s = titre;
        this.date_ds = datedeb;
        this.date_fs = datefin;
        this.temps_ds = timedeb;
        this.temps_fs = timefin;
    }

    public Suivi(int id,String user, String titre, String debut, String fin, String timedeb, String timefin) {
        this.id_s = id;
        this.client = user;
        this.titre_s = titre;
        this.date_ds = debut;
        this.date_fs = fin;
        this.temps_ds = timedeb;
        this.temps_fs = timefin;
    }

//    public Suivi(String titre,Date ){
//        
//    }
    
    public Integer getId_s() {
        return id_s;
    }

    public String getClient() {
        return client;
    }
    
    public String getTitre_s() {
        return titre_s;
    }

    public String getDate_ds() {
        return date_ds;
    }

    public String getDate_fs() {
        return date_fs;
    }

    public String getTemps_ds() {
        return temps_ds;
    }

    public String getTemps_fs() {
        return temps_fs;
    }

    @Override
    public String toString() {
        return "Suivi{" + "id_s=" + id_s + ", user=" + client + ", titre_s=" + titre_s + ", date_ds=" + date_ds + ", date_fs=" + date_fs + ", temps_ds=" + temps_ds + ", temps_fs=" + temps_fs + '}';
    }
    
    
}
